### DISCUSSION TITLE
/dye advanced permissions

### DISCUSSION THREAD

**Discussion by EkonC** [Zrips/CMI#4639] [Feature Requests]
### Description of feature/improvement request:
Add more permissions for /dye commands involving all the dye/color types.
### Detailed explanation of what it should do, when and in what situation: ###
There should be a specific permission for all colors - hex, dynamic and normal
The permissions would look something like `cmi.command.dye.[colortype].[color]` where colortype could be hex (and color would be hex code), dynamic (color would be rainbow, day, biome or health) or normal (black, blue, etc.)

Also permissions like `cmi.command.dye.clear` and `cmi.command.color.random` would be great.
### Basic explanation as to why this feature is needed and/or useful to have: ###
It would be nice to have /dye available for everyone but dynamic or hex colors for VIPs only.

_I know that this suggestion is similiar to https://github.com/Zrips/CMI/issues/2568 but I think my permission nodes are more practical._

**Comment by dkalaxdk** [comment]
That could be resolved by using Aliases? 
Just create aliases for the colours you want, and grant permissions for the individual colours you want :)

**Comment by EkonC** [comment]
Actually not quite so. How would you do it with hex codes? or so..
But thank you I may use it as a temporary solution for those dynamic colors.
